RoadNamer/Tools/RoadSelectTool.cs

```diff
@@ -70,6 +94,14 @@ protected override void OnToolUpdate()
                                     OptionsManager.SaveOptions();
                                 }
                             }
+                            else if(Event.current.type == EventType.MouseDown && Event.current.button == (int)UIMouseButton.Left)
+                            {
+                                foreach(ushort segmentId in netSegmentIds)
```

Comment:
There's differing behaviour here ( assume netSegmentIds will be valid for both single and multi segment cases ), and elsewhere ( use netSegmentId for single case, netSegmentIds for multiple cases ), netSegmentId should probably be removed or something.